stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -58,3 +64,17 @@ def _compute_procure_recommended(self):
                 qty = op._get_procure_recommended_qty(virtual_qty, op_qtys)
             op.procure_recommended_qty = qty
             op.procure_recommended_date = op._get_date_planned(qty, datetime.today())
+
+    def _quantity_in_progress(self):
+        # Overloaded to handle NewID instances.
+        # Required when this method is called from a compute method
+        if self.env.context.get("handle_newid"):
+            actual_ids = []
+            for op in self:
+                if isinstance(op.id, models.NewId):
+                    if op.id.origin:
+                        actual_ids.append(op.id.origin)
+                    # Skip NewID without origin: this case should not occurs
+            # Duplicate records having NewIds to satisfy key lookup in upcoming calls
+            self |= self.browse(actual_ids)
```

Comment:
I don't really like this. I'd propose to odoo core to split out the domain. Meanwhile, override the full method here and split the domain part.

Reply:
Checked together: I changed the fix to make it simpler